src/main/resources/patient.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS patient;
```

Comment:
please remove this,  everything about the structure should be in the same sql (for now it's not needed and used anywhere)

Reply:
remove, meaning completely delete file? or just the 1st line?

Reply:
yes, completely, you don't have a usage of this file, and also not in this phase (like having in memory), Alina is already working on creating the database structure